### DISCUSSION TITLE
How can I set the general font size of the UI ?

### DISCUSSION THREAD

**Discussion by Typhoor** [meerk40t/meerk40t#2398] [Q&A]
Using a 4k Monitor the font of the dialogs and especially the statusbar are very small.
Where can the respective font size set ?
My windows (11) display setting is 100%
The toolbar, panel and ribbon font size grow with the size of the parts, but the statusbar size is fixed vertically
Rgds Ralph

![grafik](https://github.com/meerk40t/meerk40t/assets/18496439/1ab99f9d-fc02-41d3-b82b-db0422327e08)

**Comment by tatarize** [comment]
I checked the code, we don't appear to manipulate the text size there at any point.

**Reply by tatarize** [reply]
I'll ask, we do know the monitor dip. So if it's very high res we scale the icons up. @jpirnay is there an obvious way to fix this other than just suggesting he use the help pane thing?
